jsonnet/kube-prometheus/addons/pyrra.libsonnet

```diff
@@ -22,6 +22,11 @@
     namespace:: error 'must provide namespace',
     version:: error 'must provide version',
     image: error 'must provide image',
+    resources: {
+      // TODO(paulfantom): configure limits when we have more operational data
```

Comment:
Pyrra has a built in cache for graphs. This can, by default, be up to 512MiB. We can make this configurable in Pyrra and set this, plus some extra, as it's limit.